ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerProxy.java

```diff
@@ -292,7 +295,7 @@ public RaftServerImpl getImpl(RaftGroupId groupId) throws IOException {
     return IOUtils.getFromFuture(getImplFuture(groupId), this::getId);
   }
 
-  List<RaftServerImpl> getImpls() throws IOException {
+  public List<RaftServerImpl> getImpls() throws IOException {
```

Comment:
Instead of making this function public, will it better to expose this function to `RaftServer`(though I am not sure whether it is good to expose RaftServerImpl to RaftServer)

Reply:
We should not expose RaftServerImpl to RaftServer.  Thanks for asking.